runtime/simx/vortex.cpp

```diff
@@ -154,26 +169,18 @@ class vx_device {
     }
 
     uint64_t init_pAddr = *dev_pAddr;
```

Comment:
This confuses me a bit.
I would have thought that before `need_trans()` is called we don't know whether we are dealing with a physical or virtual address.
If no translation is needed we are dealing with a physical address, otherwise we are dealing with a virtual address that requires translation.

But in this code base it seems that addresses which need translation are called p(hysical)address?